azure_functions_worker/utils/common.py

```diff
@@ -146,11 +146,15 @@ def __init__(self, file_name: str) -> None:
             f'Invalid file name: {file_name}')
 
 
-def validate_script_file_name(file_name: str):
-    # First character can be a letter, number, or underscore
-    # Following characters can be a letter, number, underscore, hyphen, or dash
-    # Ending must be .py
+def validate_script_file(function_path: Union[str, os.PathLike],
+                         file_name: str) -> (bool, bool):
+    """ Validate the script file name.
+    Returns:
+    T, T : V2 programming model
+    T, F : V2 programming model but with invalid file name
+    F, T : File path does not exist but filename is valid.
```

Comment:
There is a special case if someone actually sets it in the app setting, which could be this case and not a V1 programming model. We might want to ensure that in the future releases (TODO: Add another check if the setting was set or not and make the call based on it)

Reply:
Added a check if the app setting has been set. If it has, return T F scenario. In dispatcher changed the logs slightly for this case